.gitignore

```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2018 ArSysOp
+# Copyright (c) 2018-2020 ArSysOp
```

Comment:
Shouldn't we use comma instead of hyphen here?

Reply:
I don't think so because currently we are using Apache 2.0

Reply:
Handbook declares *copyright* statement stracture without regard to any particular license. As the project is Eclispse-oriented, I think we should follow these rules.

Reply:
agreed, please review the updated version